nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/JDBCRowMapper.java

```diff
@@ -1151,9 +1151,9 @@ protected Serializable copyHier(Serializable id, Serializable parentId, String n
     /**
      * Gets the children ids and types of a node.
      */
-    protected List<IdWithTypes> getChildrenIdsWithTypes(Serializable id, boolean onlyComplex) throws SQLException {
+    protected List<IdWithTypes> getChildrenIdsWithTypes(Serializable id, boolean onlyComplex, boolean excludeSpecialChildren) throws SQLException {
```

Comment:
This original naming isn't very clear anymore with this second argument. Could you rework things everywhere so that `onlyComplex` is renamed `excludeRegularChildren`?

Reply:
I did rename it. 
It led me to consider renaming `selectComplexChildrenIdsAndTypesSql` and `selectComplexChildrenIdsAndTypesSqlIncludeSpecialChildren` as well, but I felt it would be confusing.
Please Let me know what you think